src/main/java/ui/Ui.java

```diff
@@ -0,0 +1,47 @@
+package ui;
+
+import java.util.Scanner;
+
+/**
+ * Encapsulates a User Interface class.
+ */
+public class Ui {
+    private Scanner scanner = new Scanner(System.in);
+
+    /**
+     * Displays the welcome message.
+     */
+    public String displayWelcome() {
+        return "Hello from Bikini Bottom!\n"
+            + "____________________________________________________________\n"
+            + "Hello! I'm Spongebob\n"
+            + "What can I do for you?";
+    }
+
+    /**
+     * Displays a line.
+     */
+
+    public String displayLine() {
+        return "____________________________________________________________\n";
+    }
+
+    /**
+     * Reads the command from the user.
```

Comment:
The comments seems good and it would be better if you could add space between the description and the param description.